contents/dotfiles/Dotfiles.md

```diff
@@ -0,0 +1,231 @@
+<frontmatter>
+  title: Introduction to Dotfiles
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Dotfiles
+
+**Author(s): [Tiu Wee Han](https://github.com/tiuweehan)**<br>
+
+<box id="article-toc">
+
+* [Introduction to Dotfiles](#introduction-to-dotfiles)
+    * [What are Dotfiles?‎](#what-are-dotfiles)
+    * [Why Dotfiles?](#why-dotfiles)
+        * [Benefit: Save and Reuse Configurations](#benefit-save-and-reuse-configurations)
+        * [Benefit: Personalised Shortcuts and Commands‎](#benefit-personalised-shortcuts-and-commands‎)
+        * [Benefit: Shell Utility and Styling](#benefit-shell-utility-and-styling)‎
+    * [Getting Started](#getting-started)
+        * [Management Strategies‎](#management-strategies)
+    * [Conclusion](#conclusion)
+
+</box>
+
+### What are Dotfiles?
+Dotfiles are plain text configuration files on Unix like systems (e.g. MacOS, Linux, BSD). Dotfiles store settings of almost every application, service and tool running on your system. These files control the behavior of applications from boot to termination and everything in between.
+
+<pic src="dotfiles-logo.png" alt="Dotfiles Logo" width="45%">
+  
+  <sub>_Figure 1. Dotfiles Logo_ [(source)](https://www.twilio.com/blog/using-dotfiles-productivity-bootstrap-systems)</sub>
+
+</pic>
+
+Some common examples of dotfiles include:
+- `.profile`, `.bashrc`, `.zshrc`: scripts that run when a Shell is started. 
+- `.nanorc`, `.vimrc`, `.emacs.d`: dotfiles for configuring text editors like nano, vim and emacs respectively.
+- `.gitconfig`, `.gitignore`: dotfiles containing git configurations.
+- `.ssh/config`: dotfiles containing SSH configurations.
+
+### Why Dotfiles?
+1. **Dotfiles improve productivity.** Dotfiles allow users to tailor their environments to their needs and preferences, from general purpose configurations to project-specific commands. There are also many widely available libraries and plugins to choose and match, providing the most optimal tools to improve their productivity.
+
+1. **Feel at `$HOME` Wherever You Go.** For developers who have customised their local environments to their personal tastes, there is no place as good as `$HOME`. With proper dotfiles management, it is possible to easily port these customisations to any system, and remove them just as easily.
+
+1. **Improve Linux skills.** Setting up dotfiles is an enriching experience that will enable developers to improve command of various skills such as shell scripting and linux tools. Configuring application related tools such as `git` and `ssh` is also a good opportunity to familiarise and become experts with using these tools.
+
+#### Benefit: Save and Reuse Configurations
```

Comment:
Perhaps can place an index for each benefit and feature? aka
Benefit 1: Save ....
Feature 1.1: Git Dotfiles
Feature 1.2: SSH Dotfiles

Benefit 2: Personalised...

Reply:
Good suggestion!